src/test/java/org/xbib/elasticsearch/index/mapper/langdetect/DetectLanguageAccuracyTest.java

```diff
@@ -121,12 +121,20 @@ public void test() throws IOException {
         // Set up the detection service according to the test's parameters
         String languageSetting = DEFAULT_LANGUAGES;
         if (useAllLanguages) {
-            languageSetting = useShortProfile ? ALL_SHORT_PROFILE_LANGUAGES : ALL_DEFAULT_PROFILE_LANGUAGES;
+            // TODO: This is a bit clunky. LangdetectService should support "all" as a language setting.
```

Comment:
I figured that this is something that can be fixed in a separate PR, as it requires some changes to `LangdetectService`.